backend/cmd/app/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi/middleware"
+	"github.com/go-chi/chi/v5"
+	"github.com/ilexsor/internal/database"
+	"github.com/ilexsor/internal/handlers"
+	"github.com/ilexsor/internal/utils"
+	"github.com/joho/godotenv"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	frontendDir = http.Dir("../../../web")

```

Comment:
Когда запускаешь от корня, то будет так
```suggestion
	frontendDir = http.Dir("./web")

```